server.ts

```diff
@@ -15,12 +16,25 @@ app
   .use(helmet())
   .use(morgan("dev"))
   .use(express.json())
-  .use(express.urlencoded({ extended: true }));
+  .use(express.urlencoded({ extended: true }))
+  .use(
+    cookieSession({
+      name: "session",
+      keys: ["key1", "key2"],
```

Comment:
process.env.SESSION_SECRET instead of key1 and key2?

Reply:
Will do. This was there just for testing.